docs/data/material/components/pickers/deprecated-pickers.md

```diff
@@ -0,0 +1,19 @@
+# Date / Time pickers
+
+<p class="description">Now part of MUI X, a collection of advanced components.</p>
+
+## What is the reason to move the Pickers to MUI X?
```

Comment:
Grammar. Also, I don't think "Picker" should be capitalized since we're talking about a category rather than a specific component.

```suggestion
## Why have the picker components been moved to MUI X?
```